src/ImageStorage.php

```diff
@@ -124,7 +124,7 @@ public function saveUpload(FileUpload $upload, string $namespace, ?string $check
 
 		return new Image($this->friendly_url, $this->data_dir, $this->data_path, $identifier, [
 			'sha' => $checksum,
-			'name' => self::fixName($upload->getName()),
+			'name' => self::fixName($upload->getUntrustedName()),
```

Comment:
Could you please give me a hint why is this changed? Thanks.

Reply:
besause of deprecated of https://github.com/nette/http/blob/v3.1.5/src/Http/FileUpload.php#L67

Reply:
OK, nice, thank!